runtime/doc/options.txt

```diff
@@ -6519,10 +6518,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	|+path_extra| feature}
 	The |tagfiles()| function can be used to get a list of the file names
 	actually used.
-	If Vim was compiled with the |+emacs_tags| feature, Emacs-style tag
```

Comment:
regarding etags: there's a line which should removed in `tagsrch.txt` line 478, and one reference of `emacs_tags` in `eval.txt`.